packages/bruno-app/src/components/CodeEditor/index.js

```diff
@@ -71,7 +71,8 @@ if (!SERVER_RENDERED) {
     'bru.setNextRequest(requestName)',
     'req.disableParsingResponseJson()',
     'bru.getRequestVar(key)',
-    'bru.sleep(ms)'
+    'bru.sleep(ms)',
+    'bru.isRunner()'
```

Comment:
Can we convert this to use trailing comma to avoid having this 2 changes?

Reply:
I could also just put it before the `bru.sleep(ms)` so there's only one line changed. Though I don't think we need to discuss a single character which is added or removed from the PR.